patterns/complete_constructor.md

```diff
@@ -93,33 +97,79 @@ Thing.prototype.doYourThing = function() {
   ...
 }
 ```
-
-So, without having to read the implementation details of any of the methods, one who would like to use the Thing class would write…
-
+So, without having to read the implementation details of any of the methods, you should be able to use the Thing class like so:
 ```js
 var thing = new Thing(name);
 thing.doYourThing();
 ```
 
-vs.
```

Comment:
These changes are confusing.  I would leave them as is...

The point about how the constructor should be written is earlier.  Now we are talking about how others use the class/constructor.